app/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require('./app.js')
+const request = require('supertest')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index.js')
+
+
+
+
+beforeEach(() => seed({articleData, commentData, topicData, userData}));
+afterAll(() => db.end());
+
+describe('GET',() => {
+    describe('/api/topics', () => {
+        test('return with an array of topic objects, which have a "slug" and "description" property ', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                console.log(response)
```

Comment:
don't forget to clear out console logs when development of an endpoint is finished 🧹